components/Author/Feed/types/forYouFeedTypes.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  Paper,
+  Post,
+  parsePaper,
+  parsePost,
+} from "~/components/Document/lib/types";
+import Bounty from "~/config/types/bounty";
+import { parseContentType } from "~/config/types/contentType";
+import { Hub, parseHub } from "~/config/types/hub";
+import {
+  ID,
+  RHUser,
+  UnifiedDocument,
+  parseUser,
+} from "~/config/types/root_types";
+
+export interface ForYouFeedItem {
+  item: Paper | Post | Bounty | ForYouFeedCommentObject;
+  contentType: string;
+
+  hubs: Hub[];
+  unifiedDocument: UnifiedDocument;
+
+  userVote: number | null;
+  score: number;
+
+  createdBy: RHUser;
+  createdDate: string;
+}
+export interface ForYouFeedPaperItem extends ForYouFeedItem {
```

Comment:
Maybe we should use more generic terms like `FeedPaperItem`. Reason being is that this JSON structure will be returned in other areas of the site as well. For example, I intend to update the live feed, profile feed, ref manager feed, ... to include data that conform to this structure.